reports/tasks.py

```diff
@@ -76,6 +68,21 @@ def send_trial_instances_report(recipients=settings.TRIAL_INSTANCES_REPORT_RECIP
     end_of_last_month = (beginning_of_this_month - timedelta(days=1))
     beginning_of_last_month = end_of_last_month.replace(day=1)
 
+    # Get instances with an active betatestapplication,
+    # whose InstanceReference has an active OpenEdXAppServer
+    # and was created last month
+    instances = OpenEdXInstance.objects.exclude(
+        betatestapplication__isnull=True
+    ).filter(
+        ref_set__openedxappserver_set___is_active=True
+    ).filter(
+        ref_set__created__range=[
+            beginning_of_last_month.strftime("%Y-%m-%d"),
+            end_of_last_month.strftime("%Y-%m-%d")
+        ])
```

Comment:
@spokerman12, nit: closing parenthesis formatting.

Reply:
Changed according to previous suggestions.